packages/icons/config/fabric-icons-2.json

```diff
@@ -125,7 +122,7 @@
 			"unicode": "E940"
 		},
 		{
-			"name": "OneDrive",
+			"name": "OneDriveLogo",
```

Comment:
This rename is concerning. @mikewheaton have you ran across this in FabricCore font updates yet? Might have to do some mapping for this change to not be breaking. @Jahnp for FYI

Reply:
Great catch. We can't be renaming icons like this. Isn't there a policy to avoid this?

Reply:
Sorry--this was due to #4659 . 

There was a change for consistency that ended up going into the icons font much quicker than anticipated, which got picked up when I later updated the tool to the latest icons. Internally within the tool, we maintain a list of renamed or removed icons to prevent major breaks like this from happening--but I hadn't updated that list before this PR was opened. I've been working with @lynamemi offline on this, and I believe she'll have a new PR here soon.

Reply:
Yep! Here is the new PR with the registered alias: https://github.com/OfficeDev/office-ui-fabric-react/pull/4872